### DISCUSSION TITLE
Copilot Subscription expired earlier just 15 days not 1 month

### DISCUSSION THREAD

**Discussion by Philiptebandeke** [community/community#43610] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello my github subscription has expired after only 15 days.
I paid for it on Friday December 23rd, 2022 using a visa card and it has expired today on Sunday 8th January, 2022.
My account is [https://github.com/Philiptebandeke](url) .

Thank you.
<img width="960" alt="Screenshot_20230108_075845" src="https://user-images.githubusercontent.com/43267922/211181231-44032ce5-3d5f-413f-a577-de46b27e945c.png">
<img width="362" alt="Screenshot_20230108_075815" src="https://user-images.githubusercontent.com/43267922/211181234-c3eec9c1-5826-4ef8-95c3-6581472296c2.png">

**Comment by jongwooo** [comment]
@Philiptebandeke You can create related ticket on [GitHub Support](https://support.github.com/request) portal.

**Comment by otkrickey** [comment]
Solved: #43634
